scripts/sign-win.ps1

```
@@ -0,0 +1,48 @@
+
+$root = Resolve-Path (Join-Path $PSScriptRoot '..')
+$files = @(
+  (Join-Path $root 'release\OpenMausBot-0.1.27-setup.exe'),
+  (Join-Path $root 'release\win-unpacked\OpenMausBot.exe')
+)
+
+if ($Thumbprint) {
+  $cert = Get-ChildItem "cert:\CurrentUser\My" | Where-Object { $_.Thumbprint -eq $Thumbprint }
+  if (-not $cert) { throw "cert with thumbprint $Thumbprint not in CurrentUser\My" }
+} else {
+  $sec = ConvertTo-SecureString -String $Password -Force -AsPlainText
```

PSScriptAnalyzer (1.25.0)

[error] File 'sign-win.ps1' uses ConvertTo-SecureString with plaintext. This will expose secure information. Encrypted standard strings should be used instead.

(PSAvoidUsingConvertToSecureStringWithPlainText)